lib/oli_web/components/delivery/students/students.ex

```diff
@@ -375,16 +388,70 @@ defmodule OliWeb.Components.Delivery.Students do
       <% end %>
       <input name="role" value={@add_enrollments_selected_role} />
       <input name="section_slug" value={@section_slug} />
+      <input name="inviter" value={@inviter} />
       <button type="submit" class="hidden" />
     </.form>
     <div class="px-4">
       <p>
         Are you sure you want to enroll <%= "#{if length(@add_enrollments_emails) == 1, do: "one user", else: "#{length(@add_enrollments_emails)} users"}" %>?
       </p>
+      <.render_inviter
+        current_author={@current_author}
+        current_user={@current_user}
+        inviter={@inviter}
+        myself={@myself}
+      />
     </div>
     """
   end
 
+  attr(:current_author, :any, required: true)
+  attr(:current_user, :any, required: true)
+  attr(:inviter, :string, required: true)
+  attr(:myself, :string, required: true)
+
+  defp render_inviter(assigns) do
+    ~H"""
+    <div
+      :if={show_senders(@current_author, @current_user)}
+      class="mt-5 p-5 border-solid border-2 border-blue-400 rounded"
+    >
+      <p>You're signed with two accounts.</p>
+      <p>Please select the one to use as an inviter:</p>
+      <fieldset class="mt-2">
+        <div class="ml-2">
+          <input
+            type="radio"
+            id="author"
+            name="inviter"
+            phx-value-inviter="author"
+            phx-click="select_inviter"
+            phx-target={@myself}
+            checked={if @inviter == "author", do: true, else: false}
```

Comment:
Nit: you could just do `checked={@inviter == "author"}`

Reply:
Great! I'll do the changes then.